forge-game/src/main/java/forge/game/ability/effects/VoteEffect.java

```diff
@@ -82,16 +84,22 @@ public void resolve(final SpellAbility sa) {
         Player voter = game.getControlVote();
 
         for (final Player p : tgtPlayers) {
+            final List<Object> voteOpts = Lists.newArrayList(voteType);
             int voteAmount = p.getAdditionalVotesAmount() + 1;
             int optionalVotes = p.getAdditionalOptionalVotesAmount();
             Player realVoter = voter == null ? p : voter;
 
+            if (other) {
+                voteOpts.remove(realVoter);
+                if (voteOpts.isEmpty()) continue;
+            }
+
             Map<String, Object> params = Maps.newHashMap();
             params.put("Voter", realVoter);
             voteAmount += p.getController().chooseNumber(sa, Localizer.getInstance().getMessage("lblHowManyAdditionalVotesDoYouWant"), 0, optionalVotes, params);
 
             for (int i = 0; i < voteAmount; i++) {
-                Object result = realVoter.getController().vote(sa, host + " " + Localizer.getInstance().getMessage("lblVote") + ":", voteType, votes, p, sa.hasParam("UpTo"));
+              Object result = realVoter.getController().vote(sa, host + " " + Localizer.getInstance().getMessage("lblVote") + ":", voteType, votes, p, sa.hasParam("UpTo"));
```

Comment:
shouldn't this use voteOpts now?

Reply:
I had goofed something up in a merge conflict resolution